manuscript/4.0authentication.md

```diff
@@ -55,174 +55,174 @@ While generating a new session, we need to check if a sessions is already active
 Till now we never used any third party library or a framework in this book, this is for the first time that we are doing so, we better use libraries for handling sessions, since security is the primary aspect of any web application it is wiser to use existing (and good) packages to manage security.
 
 Path: `~/Tasks/sessions/sessions.go`
-
-    package sessions
-
-    import (
-        "net/http"
-
-        "github.com/gorilla/sessions"
-    )
-
-    //Store the cookie store which is going to store session data in the cookie
-    var Store = sessions.NewCookieStore([]byte("secret-password"))
-
-    //IsLoggedIn will check if the user has an active session and return True
-    func IsLoggedIn(r *http.Request) bool {
-        session, _ := Store.Get(r, "session")
-        if session.Values["loggedin"] == "true" {
-            return true
-        }
-        return false
-    }
-
+```golang
```

Comment:
When using syntax fences you should not indent everything four spaces. It automatically creates a syntax block that makes it look indented. 

Please see the markdown preview for an example of this.